### DISCUSSION TITLE
How would I disable damage for this?

### DISCUSSION THREAD

**Discussion by AVeryTemporaryUser** [Jcw87/c2-sans-fight#137] [Q&A]
Backstory, one of my friends wanted me to make a custom attack for another friend to beat, I want to make it absolutely painful for him, but I can't beat the attack myself (mostly because I am absolutely terrible at dodging, no matter the game), so how do I disable damage for testing?

**Comment by dl-tg** [comment]
[No damage](https://github.com/Jcw87/c2-sans-fight/pull/72)
